src/ui/add_column.rs

```diff
@@ -276,9 +279,22 @@ impl<'a> AddColumnView<'a> {
     }
 
     fn get_base_options(&self) -> Vec<ColumnOptionData> {
+        let default_locale: LanguageIdentifier = "en-US".parse().expect("Parsing failed.");
+        let resources = vec!["notedeck.ftl".into()];
+        let bundle = self.res_mgr
+            .get_bundle(
+                vec![default_locale],
+                resources,
+            )
+            .expect("Could not get bundle");
+        let mut errors = vec![];
+        let msg = bundle.get_message("universe-title").expect("Message exists");
+        let pattern = msg.value().expect("Message has a value");
+        let value: &'static mut str = bundle.format_pattern(pattern, None, &mut errors).to_string().leak();
```

Comment:
This `leak` intuitively seems wrong to me, but I couldn't get passed compilation errors without it. `title` in `ColumnOptionData` is static and I didn't know how to get the string from the fluent library there in any other way. Looking for feedback here.

Reply:
I think we just need to cache the bundle in our app state and fix some of these string lifetimes and we should be good to go. I'll take a pass at it. thanks for getting this started